EmailNotifications/Email.cs

```diff
@@ -88,11 +97,16 @@ public void sendEmail()
                 // SEND IN HTML FORMAT (comment this line to send plain text).
                 message.IsBodyHtml = true ;
                 message.Body = outS;
+                string current =  Directory.GetCurrentDirectory();
+                string sFile = current + @"\logo.PNG";
```

Comment:
I can see that you're calling this image from your current directory.
Next time think of using cdn to speed up the retrieval of this image.

Reply:
alright thank you